pkg/gcv/validator.go

```diff
@@ -86,14 +101,28 @@ func NewValidatorConfig(policyPaths []string, policyLibraryPath string) (*config
 func newCFClient(
 	targetHandler cfclient.TargetHandler,
 	templates []*cftemplates.ConstraintTemplate,
-	constraints []*unstructured.Unstructured) (
+	constraints []*unstructured.Unstructured,
+	args ...Option) (
 	*cfclient.Client, error) {
-	driver := local.New(local.Tracing(false))
-	backend, err := cfclient.NewBackend(cfclient.Driver(driver))
+	options := &initOptions{}
+	for _, arg := range args {
+		arg(options)
+	}
+
+	defaultDriverArgs := []local.Arg{local.Tracing(false)}
```

Comment:
You can set up the initOptions with this literal so you don't need the append.   Do this for all three.